src/main/resources/application-docs.yml

```
@@ -0,0 +1,20 @@
+springdoc:
+  default-consumes-media-type: application/json;charset=UTF-8
+  default-produces-media-type: application/json;charset=UTF-8
+  swagger-ui:
+    path: /swagger-ui
+    disable-swagger-default-url: true
+    display-request-duration: true
+    tags-sorter: alpha
+    operations-sorter: alpha
+    doc-expansion: none
+    syntax-highlight:
+      theme: none
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)